devito/core/gpu_openmp.py

```diff
@@ -369,7 +460,8 @@ def _make_passes_mapper(cls, **kwargs):
             'optcomms': partial(optimize_halospots),
             'openmp': partial(ompizer.make_parallel),
             'mpi': partial(mpiize, mode=options['mpi']),
-            'prodders': partial(hoist_prodders)
+            'prodders': partial(hoist_prodders),
+            'gpu-direct': partial(mpi_gpu_direct)
```

Comment:
you also have to add `'gpu-direct' here: https://github.com/devitocodes/devito/pull/1363/files#diff-471f90b5151a59238f0735766abbda55R448

Reply:
I'm not sure I understood it. Do you mean adding
```

def _mpi_gpu_direct(self, *args):
        return
```
after line 454 of `devito/mpi/routines.py`?

Reply:
sorry, I mean, you have to add it here: https://github.com/devitocodes/devito/blob/master/devito/core/gpu_openmp.py#L360, next to `'prodders'` so that the compiler recognizes it

Reply:
NP